include/tvm/meta_schedule/database.h

```diff
@@ -357,6 +357,18 @@ class Database : public runtime::ObjectRef {
    */
   TVM_DLL static Database JSONDatabase(String path_workload, String path_tuning_record,
                                        bool allow_missing);
+  /*!
+   * \brief Create a database merged from multiple databases.
+   * \param preferred The preferred database. If the preferred database responses to a query,
+   * all other databases will be ignored.
```

Comment:
Nit, maybe better note that `preferred` is an array of preferred databases and the preferred databases are given in specific order so that any database in the are preferred than the databases after it.

Reply:
Good catch!